### PR TITLE
Write Lesson, Module and Task Builders + testcases

### PR DESCRIPTION
**Pull request by UncleGrandpa925**

### CODE REVIEW

**Comment by UncleGrandpa925** [comment]
Hi @simonteozw . Actually this PR is kinda simple and there is not much to review, yet I still request your review so that you can know about the newly added test skeleton! Thank you!

File: src/test/java/trackitnus/logic/commands/module/AddModuleCommandIntegrationTest.java [hunk 1]
```diff
@@ -44,10 +43,4 @@ public void execute_duplicateModule_throwsCommandException() {
         assertCommandFailure(new AddModuleCommand(moduleInList), model, Messages.MESSAGE_DUPLICATE_MODULE);
     }
 
-    @Test
-    public void execute_duplicateModuleCode_throwsCommandException() {
-        Module validModule = new Module(new Code("CS1101S"), new Name("Sample"));
-        assertCommandFailure(new AddModuleCommand(validModule), model, Messages.MESSAGE_DUPLICATE_MODULE);
-    }
-
```
Comment (simonteozw) [inline src/test/java/trackitnus/logic/commands/module/AddModuleCommandIntegrationTest.java:52]: Hmm why did you remove this test case?

**Review by simonteozw — COMMENTED** [review]
Looks good! Just 1 comment

File: src/test/java/trackitnus/logic/commands/module/AddModuleCommandIntegrationTest.java (hunk 1 above)
Comment (UncleGrandpa925) [reply src/test/java/trackitnus/logic/commands/module/AddModuleCommandIntegrationTest.java:52]: because I have added a very similar testcase to line 41-45

Comment (UncleGrandpa925) [reply src/test/java/trackitnus/logic/commands/module/AddModuleCommandIntegrationTest.java:52]: because I have added a very similar testcase to line 41-45 
image

**Review by simonteozw — APPROVED** [review]
LGTM
